### PR TITLE
Remove lang3 usage

### PR DESCRIPTION
**Pull request by KingOfSquares**
Necessary for #54

### CODE REVIEW

File: core/src/main/java/tc/oc/pgm/killreward/KillRewardMatchModule.java [hunk 1]
```diff
@@ -34,19 +33,19 @@
 @ListenerScope(MatchScope.RUNNING)
 public class KillRewardMatchModule implements MatchModule, Listener {
   private final Match match;
-  private final Map<UUID, MutableInt> killStreaks;
+  private final Map<UUID, Integer> killStreaks; // TODO: switch to fastutil Object2IntMap??
```
Comment (Pablete1234) [inline core/src/main/java/tc/oc/pgm/killreward/KillRewardMatchModule.java]: Normal map is fine

File: core/src/main/java/tc/oc/pgm/map/MapInfoImpl.java [hunk 2]
```diff
@@ -201,7 +200,7 @@ public boolean equals(Object obj) {
 
   @Override
   public String toString() {
-    return new ToStringBuilder(this).append("id", getId()).build();
+    return "MapInfoImpl{" + "id='" + id + '\'' + ", version=" + version + '}';
```
Comment (Pablete1234) [inline core/src/main/java/tc/oc/pgm/map/MapInfoImpl.java]: name or something?

File: core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java [hunk 3]
```diff
@@ -54,7 +51,7 @@
 import tc.oc.pgm.util.named.NameStyle;
 import tc.oc.pgm.util.nms.NMSHacks;
 
-public class MatchPlayerImpl implements MatchPlayer, Comparable<MatchPlayer> {
+public class MatchPlayerImpl implements MatchPlayer {
```
Comment (Pablete1234) [inline core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java:54]: This looks like a dangerous change

**Review by Pablete1234 — CHANGES_REQUESTED** [review]

File: core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java (hunk 3 above)
Comment (KingOfSquares) [reply core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java:54]: The method was not used anywhere I could find, it was especially weird that it was scoped to `MatchPlayerImpl` and not `MatchPlayer`

Comment (Pablete1234) [reply core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java:54]: That's all fine until there's somewhere in the code that uses a TreeSet<MatchPlayer> or similar and you're doomed

Comment (KingOfSquares) [reply core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java:54]: `Match` does not implement `Comparable`, do you think that is something to include back as well?

Comment (Pablete1234) [reply core/src/main/java/tc/oc/pgm/match/MatchPlayerImpl.java:54]: It probably should for ease of use, but the example of something that could break doesn't necessarily need it to work

**Review by Electroid — APPROVED** [review]

**Comment by Electroid** [comment]
Can you re-open this?

**Pull request merged**